.travis.yml

```diff
@@ -0,0 +1,18 @@
+script:
+- echo "hello world"
+- docker build --target development --tag todo-app:dev .
+- docker build --target production --tag todo-app:prod .
+- docker build --target test --tag todo-app:test .
+- docker run todo-app:test tests/test_viewmodel.py
+- docker run --env-file .env.test todo-app:test tests/test_app.py
+- docker run --env TRELLO_BOARD_NAME_CREATE=E2E_test_board --env TRELLO_KEY=${TRELLO_KEY}
```

Comment:
Docker has a short syntax when you're passing environment variables that have the same name inside & outside the container;
`docker run TRELLO_BOARD_NAME_CREATE=E2E_test_board --env TRELLO_KEY --env TRELLO_TOKEN ...`

Reply:
Thanks - I have added these changes.